### PR TITLE
[WIP] Update node-reference

### PR DESCRIPTION
**Pull request by natowi**
Add new nodes, update old nodes.

[ x ] add node summary to all new nodes
[ x ] update old nodes (FeatureExtraction, Meshing) (add params table)
[ x ] Fix FeatureMatching Table (too small)
[ x ] Add new nodes
[ x ] fix most build warnings
[ x ] note that some features/settings/nodes are only available in the develop

ToDo for next PR:
[  ] update panorama related nodes, fix formatting in a few files, add new node graph, 

[  ] fix new build log errors

[  ] check https://github.com/alicevision/meshroom/pull/768 and https://github.com/alicevision/meshroom/pull/731

### CODE REVIEW

File: source/node-reference/node-reference.rst [hunk 1]
```diff
@@ -5,14 +5,19 @@ List with all the nodes
...
+   
+ 
 
 .. toctree::
    :maxdepth: 1
    :hidden:
    :caption: Node Reference
    
    nodes/CameraCalibration
+   nodes/CameraDownscale
```
Comment (fabiencastan) [inline source/node-reference/node-reference.rst:20]: This has been moved in an "ImageProcessing" node.

File: source/node-reference/node-reference.rst [hunk 2]
```diff
@@ -22,9 +27,12 @@ List with all the nodes
    nodes/DepthMapFilter
    nodes/ExportAnimatedCamera
    nodes/ExportColoredPointCloud
+   nodes/ExportMatches
    nodes/ExportMaya
    nodes/FeatureExtraction
    nodes/FeatureMatching
+   nodes/GlobalSfM
+   nodes/HDRIstitching
```
Comment (fabiencastan) [inline source/node-reference/node-reference.rst:35]: This has been removed. We now use "Panorama*" nodes.

Comment (natowi) [reply source/node-reference/node-reference.rst:35]: So panoramaInit, panoramaEstimation, panoramaWarping, panoramaCompositing ...

**Comment by simogasp** [comment]
is it still WIP or can we merge?

**Comment by natowi** [comment]
@simogasp I need to do a final review of all the files. I think it will be ready to be merged at the end of the month.

**Comment by simogasp** [comment]
cool! i was just checking the status, thanks!

**Comment by natowi** [comment]
@simogasp you can merge this PR now. I will fix the other stuff and add new changes in a new PR.

**Comment by natowi** [comment]
Can we merge this now?

**Pull request merged**